tokio/src/runtime/queue.rs

```diff
@@ -93,6 +93,13 @@ impl<T> Local<T> {
         !self.inner.is_empty()
     }
 
+    /// Returns false if there are any entries in the queue
+    /// Separate to is_stealable so that refactors of is_stealable to "protect"
+    /// some tasks from stealing won't affect this
+    pub(super) fn has_tasks(&self) -> bool {
```

Comment:
When you have an explicit line break in a comment, you should leave an empty line between the two lines because that's how markdown works. Also, you're missing a period or two.